packages/plugin-npm/sources/npmHttpUtils.ts

```diff
@@ -66,7 +70,12 @@ export async function get(path: string, {configuration, headers, ident, authType
     headers = {...headers, authorization: auth};
 
   try {
-    return await httpUtils.get(path.charAt(0) === `/` ? `${registry}${path}` : path, {configuration, headers, ...rest});
+    const url = path.charAt(0) === `/` ? `${registry}${path}` : path;
+
+    if (typeof rest.reportInfo !== `undefined`)
+      rest.reportInfo(MessageName.UNNAMED, `Fetching ${url}`);
```

Comment:
If we decide to log from here, I am not sure what `MessageName` to use. Is `UNNAMED` sufficient? Should I create a new one?

Reply:
I guess I should also deconstruct `reportInfo` from `rest` instead of getting it off of `rest`. I just wanted to highlight the fundamentals of this change.

Also, there might be other code that calls this where we would also want to pass down `reportInfo` -- that way we could get a comprehensive audit trail in the logs of all the packages we attempted to pull.

This might help in investigations into dependency confusion attacks as well.